src/scope/spout/receiver.py

```diff
@@ -0,0 +1,265 @@
+"""
+Spout Receiver - Receives textures from Spout senders.
+
+This module provides a simple interface for receiving textures from
+Spout-compatible applications like TouchDesigner, Resolume, etc.
+
+Note: The Python SpoutGL wrapper API differs from the C++ SDK.
+See: https://github.com/jlai/Python-SpoutGL
+"""
+
+import logging
+
+import numpy as np
+
+logger = logging.getLogger(__name__)
+
+# Check if SpoutGL is available
+try:
+    import SpoutGL
+
+    SPOUT_AVAILABLE = True
```

Comment:
As mentioned in another comment feels like better behavior is raising import error here and having FrameProcessor take care of it as opposed to trying to silently continue with just a warning. This could result in a cleaner code eg not having to sprinkle all those checks on whether SpoutGL is available.